script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -98,7 +98,7 @@ Hisui bows and leaves the room.
 [sha:8acf5b152d3170b9020122af3087dd7674403e8b]{
 -- Page 8, Offset 16584.
 -- 心なしか、普段よりややお辞儀にキレがなかったような。
-Perhaps it's just me, but her bow didn't seem as sharp as usual.
+Perhaps it's just me, but her bow didn't seem as rigid as usual.
```

Comment:
I'd keep sharp here, I think it's a more common adj for this use. "Rigid" also seems more uncomfortable. 
![2022-01-21-210950_1282x455_scrot](https://user-images.githubusercontent.com/2476799/150620827-821533fa-9489-4ec3-859f-c5d9f6c275a2.png)